homeassistant/helpers/intent.py

```diff
@@ -16,6 +16,8 @@
 INTENT_TURN_OFF = 'HassTurnOff'
 INTENT_TURN_ON = 'HassTurnOn'
 INTENT_TOGGLE = 'HassToggle'
+INTENT_OPEN_COVER = 'HassOpenCover'
```

Comment:
Let's keep them defined inside `cover/__init__.py`

Reply:
Hmm, the issue I see is that we then need to import all the modules that have intents when we load conversation. Not a huge deal and I can't see another obvious place unless we move it to const or somesuch.